openedx/features/content_tagging/tests/test_tasks.py

```diff
@@ -0,0 +1,240 @@
+"""
+Test for auto-tagging content
+"""
+from __future__ import annotations
+
+from unittest.mock import patch
+
+from django.core.management import call_command
+from django.test import override_settings
+from edx_toggles.toggles.testutils import override_waffle_flag
+from openedx_tagging.core.tagging.models import ObjectTag, Tag, Taxonomy
+from organizations.models import Organization
+
+from common.djangoapps.student.tests.factories import UserFactory
+from openedx.core.djangolib.testing.utils import skip_unless_cms
+from xmodule.modulestore.tests.django_utils import TEST_DATA_MIXED_MODULESTORE, ModuleStoreTestCase
+
+from .. import api
+from ..models import ContentLanguageTaxonomy, TaxonomyOrg
+from ..toggles import CONTENT_TAGGING_AUTO
+
+LANGUAGE_TAXONOMY_ID = -1
+
+
+@skip_unless_cms  # Auto-tagging is only available in the CMS
```

Comment:
Hm..
Do you know a way to include these tests in the CMS test group? The view is not enabled in LMS, so it will fail there.

Reply:
> Do you know a way to include these tests in the CMS test group? The view is not enabled in LMS, so it will fail there.

Oh I understand, I haven't found an example of it, I'm not sure if it's a good practice. @bradenmacdonald could you help us with this?

Reply:
@ChrisChV good catch - thanks!

Unfortunately it seems like [the whole `openedx/features/` directory is only tested as LMS on CI](https://github.com/openedx/edx-platform/blob/db459761d86ae1379344d6fcb8eda048674b8f80/.github/workflows/unit-test-shards.json#L157). What you'll need to do is after this PR merges, open a second PR to move this whole `content_tagging` app from `openedx/features/` to `openedx/core/djangoapps/content_tagging`. Then, you can edit `unit-test-shards.json` and put it into shard `openedx-3` or `openedx-4` (whichever is currently running faster on the CI).